CHANGELOG.md

```diff
@@ -4,7 +4,7 @@
 
 ### Documentation
 
-* [ENHANCEMENT] Improve `MimirIngesterReachingTenantsLimit` runbook. #4744
+* [ENHANCEMENT] Improve `MimirIngesterReachingTenantsLimit` runbook. #4744 #4752
```

Comment:
Is there any improvement of this runbook after merging this PR? If not, we should remove this changelog entry too :)